konrad/cloud.py

```diff
@@ -263,6 +278,51 @@ def __init__(self, z, cloud_fraction, mass_water, mass_ice,
             droplet_radius=droplet_radius
         )
 
+    def __add__(self, other):
+        """Define the superposition of two clouds in a layer."""
+        if not np.allclose(self.z, other.z):
+            raise ValueError('Incompatible `z` grids.')
+
+        cldf = np.max(
+            (self.cloud_area_fraction_in_atmosphere_layer,
+             other.cloud_area_fraction_in_atmosphere_layer),
+            axis=0,
+        ).clip(min=0, max=1)
+
+        iwc = np.max(
+            (self.mass_content_of_cloud_ice_in_atmosphere_layer,
```

Comment:
Can we please not use abbreviations :( I know in this case it's kind of obvious what they are, but I'm so fed up of variable names that don't make sense.

Reply:
I should be able to fit the full names into the lines 😉